spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-client-catalog-watcher/src/test/java/org/springframework/cloud/kubernetes/client/catalog/KubernetesClientCatalogWatchNamespacesIT.java

```diff
@@ -85,6 +85,7 @@ void beforeEach() {
 
 	@AfterEach
 	void afterEach() {
+		util.deleteClusterWide(NAMESPACE_DEFAULT, Set.of(NAMESPACE_A, NAMESPACE_B));
```

Comment:
small addition here not related to the main changes per-se. clean-up after ourselves, nothing more.